test/studies/amr/lib/amr/AMRHierarchy_def.chpl

```diff
@@ -572,7 +573,7 @@ class PhysicalBoundary
   //| >    constructor    | >
```

Comment:
I'd be tempted to change this and the one after the initializer to say "initializer" instead of "constructor" but that's really nit-picky